webapp/channels/src/components/edit_post/edit_post.tsx

```diff
@@ -486,7 +486,7 @@ const EditPost = ({editingPost, actions, canEditPost, config, channelId, draft,
         >
             <Textbox
                 tabIndex={0}
-                rootId={editingPost.post ? Utils.getRootId(editingPost.post) : ''}
+                rootId={editingPost.post ? editingPost.post.root_id : ''}
```

Comment:
Yeah, it would be a slight change. I feel like the previous behaviour is actually wrong, but looking more at the code though, the `AdvancedTextEditor` actually just passes `rootId={postId}` which I also would not expect. Something is really fishy with this code, but to avoid breaking things, I'll revert that change in logic for the time being